### ISSUE TITLE
Travis build broken on OS X: ocamlrun problem

### ISSUE THREAD

**Issue by yallop** [ocaml/opam-repository#4504]
See e.g. [here](https://travis-ci.org/ocaml/opam-repository/jobs/71682679) or [here](https://travis-ci.org/ocaml/opam-repository/jobs/71627875):
```
.travis-ci.sh: /usr/local/bin/ocaml: @@HOMEBREW_PREFIX@@/bin/ocamlrun: bad interpreter: No such file or directory
```

**Comment by AltGr** [comment]
Looks like a bug in homebrew (!?) -- the `/usr/local/bin/ocaml` it generated has, it seems, a `#!@@HOMEBREW_PREFIX@@/bin/ocamlrun` header where some replacement obviously failed to take place.

**Comment by avsm** [comment]
Is this still happening? There was a short period of breakage from homebrew/homebrew#41432 that's now fixed in homebrew/homebrew#41463

**Comment by yallop** [comment]
It appears fixed in recent builds ([e.g.](https://travis-ci.org/ocaml/opam-repository/jobs/72431947)).  I'll close.  Thanks!
